src/webapp/pages/edit/EditPage.tsx

```diff
@@ -11,14 +11,22 @@ import { DhisPage } from "../dhis/DhisPage";
 
 export interface EditPageProps {
     edit: boolean;
+    clone: boolean;
 }
 
-export const EditPage: React.FC<EditPageProps> = ({ edit }) => {
+const cancelMap = {
+    edit: "editing",
+    clone: "cloning",
+    create: "creation",
+};
```

Comment:
Since the possible UI configurations is growing we could refactor the boolean logic to an union instead:

```
export type EditPageState = "edit" | "clone" | "create";


```